parser/Builder.cc

```diff
@@ -1311,6 +1320,22 @@ class Builder::Impl {
                                  make_unique<Symbol>(keyLoc, gs_.enterNameUTF8(key->view())), std::move(value));
     }
 
+    unique_ptr<Node> pair_label(const token *key) {
+        unique_ptr<Node> value;
+        if (islower(key->asString().at(0))) {
```

Comment:
```suggestion
        if (islower(key->view()[0])) {
```

`view` provides the `string_view` internally. `asString` allocates a new string.